src/vivarium_census_prl_synth_pop/data/generate_business_names.py

```diff
@@ -0,0 +1,119 @@
+import sys
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+from loguru import logger
+
+from vivarium_census_prl_synth_pop.constants import data_values, paths
+
+
+def generate_business_names_data(n_total_names: str):
```

Comment:
To clarify, you're adding this functionality here but it's not actually being used anywhere yet? Why are you adding it now then?

Reply:
Yes.  This is to generate a list of random names.  Its going to be a big list so we want to keep it separate from the artifact.  There is also a good chance since we do not need it during the actual simulation (we will identify employers by the employer_id column) we may add them later in some type of post-processing step.